resources/lang/en/subscription.php

```diff
@@ -4,6 +4,7 @@
     'benefits'  => [
         'boosters'  => 'campaign boosters',
         'main'      => 'Subscribe to Kanka to unlock higher image uploads, an ad-free experience, :boosters and :more. We use :stripe to handle all billing, with no credit card information stored or transiting through our servers.',
+        '2fa'       => 'Subscribe to Kanka to unlock two factor authentication, higher image uploads, an ad-free experience, :boosters and :more.',
```

Comment:
The 2FA benefit should be in `callout.php` instead of `subscription.php`, as it's specific to a unique feature. Have it as `callout.subscribe.pitch-2fa` => `'Subscribe to unlock secured logins with two factor authentication.'`